### PR TITLE
case BUGZ-599: CREATE app will now recognize newer-style baked models and skyboxes as baked in entity-list

### PR DESCRIPTION
**Pull request by sethalves**
- CREATE app will now recognize newer-style baked models and skyboxes as baked in entity-list

https://highfidelity.atlassian.net/browse/BUGZ-599

### CODE REVIEW

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/19858/HighFidelity-Beta-PR15713-22d602a.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/19858/HighFidelity-Beta-Interface-PR15713-22d602a.dmg

**Comment by hifi-gustavo** [comment]
Android build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/19858/HighFidelity-Beta-PR15713-22d602a.apk).  Quest build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/19858/HighFidelity-Quest-Beta-PR15713-22d602a.apk)

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/19858/HighFidelity-Beta-PR15713-22d602a.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/19858/HighFidelity-Beta-Interface-PR15713-22d602a.exe

File: scripts/system/libraries/entityList.js [hunk 1]
```diff
@@ -164,7 +178,8 @@ EntityListTool = function(shouldUseEditTabletApp) {
             var cameraPosition = Camera.position;
             PROFILE("getMultipleProperties", function () {
                 var multipleProperties = Entities.getMultipleEntityProperties(ids, ['name', 'type', 'locked',
-                    'visible', 'renderInfo', 'modelURL', 'materialURL', 'imageURL', 'script', 'certificateID']);
+                    'visible', 'renderInfo', 'modelURL', 'materialURL', 'imageURL', 'script', 'certificateID',
+                    'skybox.url', 'ambientLight.url']);
```
Comment (SamGondelman) [inline scripts/system/libraries/entityList.js]: Should we merge this now or wait until the ambientLight.url issue is resolved?

Comment (sethalves) [reply scripts/system/libraries/entityList.js]: well... I don't think anyone is yet attempting to fix that, so... when they do, we should update this also.

Comment (SamGondelman) [reply scripts/system/libraries/entityList.js]: ok I'll leave a comment on this ticket: https://highfidelity.atlassian.net/browse/BUGZ-603

**Pull request merged**